xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/test/java/org/xwiki/refactoring/internal/job/DeleteJobTest.java

```diff
@@ -60,6 +62,10 @@
     @Rule
     public MockitoComponentMockingRule<Job> mocker = new MockitoComponentMockingRule<Job>(DeleteJob.class);
```

Comment:
Not sure how difficult it is, but you should refactor it to a ComponentTest.

Reply:
Test class migrated https://github.com/xwiki/xwiki-platform/pull/1369/files#diff-31414dcec2cad13ecebcd1afb29d5689R72